### PR TITLE
README and configuration cleanup

### PR DESCRIPTION
**Pull request by Jatc252**
Cleaned up the README and hopefully made the configuration a bit nicer. This README is also intended to replace the one on GitHub and Thunderstore.

### CODE REVIEW

File: Plugin.cs [hunk 1]
```diff
@@ -51,10 +51,10 @@ public class Plugin : BaseUnityPlugin
...
         public void Awake()
         {
-            AIStart = Config.Bind("General", "AI Start", "random", "The AI option to use. (tree, standard, or random)");
-            walkSpeed = Config.Bind("General", "Walk Speed", 3.5f, "Walking speed.");
-            runSpeed = Config.Bind("General", "Run Speed", 7.0f, "Running speed.");
-            Levels = Config.Bind("General", "Levels", "VowLevel:100;MarchLevel:100", "Moons that it will spawn on. Format: \"MoonName:Weight;MoonName2:Weight2\". All will work as a moon name");
+            AIStart = Config.Bind("General", "AI Start Animation", "random", "Which animation Siren Head will spawn in");
+            walkSpeed = Config.Bind("General", "Siren Head Walk Speed", 3.5f, "Walking speed of Siren Head");
+            runSpeed = Config.Bind("General", "Siren Head Run Speed", 7.0f, "Running speed of Siren Head");
+            Levels = Config.Bind("General", "Moons", "VowLevel:100;MarchLevel:100", "Moons that Siren Head will spawn on. Format as: \"MoonName:SpawnWeight\". Check README for more info.");
```
Comment (Jatc252) [inline Plugin.cs:57]: Not sure if it's better to say 'check README' or actually have the information here

File: README.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+# Credits
+
+* Model from [Siren Head by wollabebes on Sketchfab](https://sketchfab.com/3d-models/siren-head-e2a7eca5bc814b6ca5b2d2e85f478360)
+* Alert sounds: @birdekek and @spinmaster6701 on Discord
+* Animation: @dotflare on Discord
+* Step sounds: @spinmaster6701 on Discord
+* Walkie sounds: @spinmaster6701 on Discord
+* Terminal entry: @starstruck1234 and @dotflare on Discord
+
+The assets used in the mod can be found [here in this GitHub repository](https://github.com/Ccode-lang/LethalCompanyUnityTemplate-SirenHead).
```
Comment (Jatc252) [inline README.md:28]: Could the end (l28) be worded better?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+* AI Start Animation: Which animation Siren Head will spawn in. Options: `tree`, `standard`, `random`
+* Siren Head Walk Speed: Walking speed of Siren Head. Options: float value, default `3.5`
+* Siren Head Run Speed: Running speed of Siren Head. Options: float value, default `7.0`
+* Moons: Moons that Siren Head will spawn on. Format as: `MoonName:SpawnWeight`, for example `VowLevel:50` or `VowLevel:100;MarchLevel:100`. To spawn on all moons, `All:SpawnWeight` (includes modded moons). See below for more options.
+
+## Moons
+
+Uses [LethalLib.Modules.Levels.LevelTypes](https://github.com/EvaisaDev/LethalLib/blob/main/LethalLib/Modules/Levels.cs#L15):
+
+`ExperimentationLevel`, `AssuranceLevel`, `VowLevel`, `OffenseLevel`, `MarchLevel`, `RendLevel`, `DineLevel`, `TitanLevel`, `AdamanceLevel`, `ArtificeLevel`, `EmbrionLevel`, `Vanilla`: only the moons included with the base game, `Modded`: only modded moons, `All`: all possible moons in game.
```
Comment (Jatc252) [inline README.md:17]: Is there a better way to lay out the list of levels?

**Comment by Ccode-lang** [comment]
This looks great! Thanks for the PR.

**Pull request merged**